### ISSUE TITLE
Design inconsistencies and ambiguities

### ISSUE THREAD

**Issue by martinbagshaw** [yalla-coop/curenetics#56]
From doing the initial ui setup, I think there are a few inconsistencies we best clear up, and the user flow seems to be a bit hard to follow, I will do my best to summarise below:

[Figma design here](https://www.figma.com/file/BaIHfKxJN8zhXlVoTxlTJu7B/CURENETICS2?node-id=47%3A1181)

**Back buttons and breadcrumbs**
- We have a combinations of arrows, chevrons, uppercase and lowercase text, and purple and grey colours that have no real correlation between small and large screen sizes, or location within the app. Best to pick to one style, and stay to it.

<img width="140" alt="Screenshot 2019-09-29 at 17 39 02" src="https://user-images.githubusercontent.com/28146666/65835610-75e4ee80-e2e0-11e9-8430-ea7158a6ca0b.png">
<img width="292" alt="Screenshot 2019-09-29 at 17 39 16" src="https://user-images.githubusercontent.com/28146666/65835611-75e4ee80-e2e0-11e9-9f1e-8c1c2c7e428e.png">
<img width="216" alt="Screenshot 2019-09-29 at 17 39 59" src="https://user-images.githubusercontent.com/28146666/65835612-767d8500-e2e0-11e9-8e25-336293282578.png">
<img width="279" alt="Screenshot 2019-09-29 at 17 40 12" src="https://user-images.githubusercontent.com/28146666/65835613-767d8500-e2e0-11e9-93d4-8bcc11b315ee.png">

**icon button styles**
- would be nicer to get these two aligned (same font size and colour)
<img width="300" alt="Screenshot 2019-09-29 at 17 44 30" src="https://user-images.githubusercontent.com/28146666/65835660-f146a000-e2e0-11e9-84be-8f9287620a9a.png">
<img width="441" alt="Screenshot 2019-09-29 at 17 44 42" src="https://user-images.githubusercontent.com/28146666/65835661-f146a000-e2e0-11e9-8416-a2d34dd22056.png">

**Adding options to a menu**
- So far, we have 'about' in the menu in the top right (on desktop). Should we also include 'upload' and 'manual entry' here to make all options more apparent?

**pdf upload and manual entry combination**
- Is this something we can support? It could be a bit of a frustrating user experience to lose all results if wanting to combine both manual entry and pdf upload

**matches index (screen 13)**
- What does this mean? Are trials sorted by best match? If so, should we let the user know this
- What type of sort do the sorting arrows perform? Could you select the option you want to sort by from a dropdown instead?

<img width="418" alt="Screenshot 2019-09-29 at 17 57 34" src="https://user-images.githubusercontent.com/28146666/65835870-b003bf80-e2e3-11e9-8d92-d34ed269b38a.png">

**clinical trials details (screen 14)**
- Why do we have an input field to tell us the name of the file we are getting trials back for? Should this not be a heading or plain text? Also, what if we manually entered the data, would we say 'entry 10' instead?
- In order to not take up a huge amount of room on the page, we should have expandable cards that are collapsed by default, and expand to the size shown when clicked
- green colour used by the table on white probably isn't going to be very accessible. It could also be confused / confounded with 'potentially eligible' and 'nearly eligible' instead. Perhaps giving the whole result card a top border with the eligibility colour and status text would be a better solution.
- In each card, do we not want to export the single trial with the button, rather than 'all trials'?

**losing data after submission **
- I understand we aren't storing data on the app for confidentially / safeguarding concerns, also the extra bell and whistle of adding login functionality, **BUT** are the below alerts not somewhat contradictory?
<img width="481" alt="Screenshot 2019-09-29 at 18 09 00" src="https://user-images.githubusercontent.com/28146666/65835968-d2e2a380-e2e4-11e9-866e-91ffa3228a11.png">
<img width="415" alt="Screenshot 2019-09-29 at 18 08 01" src="https://user-images.githubusercontent.com/28146666/65835969-dc6c0b80-e2e4-11e9-9519-2c16b98be4cb.png">
- I would suggest re-wording of the first modal to say "Data you have entered or uploaded will be wiped when you click the 'match clinical trials' button", and "results will be wiped when you close the app"
- In addition, results and uploads / entries will also be wiped if a user uses the forward / back buttons in the browser _unless_ we
a) find a way to autosave their data as they enter it, and 
b) save references to their results where they are stored on the API. I would suggest doing this, as having information for say 100 trials saved in localStorage will likely put a greater performance load on the user's browser.

**Comment by thejoefriel** [comment]
Hey, a number of these changes have been discussed and incorporated so closing this issue for now. @mr-bagglesworth let me know if you want to run through any of the other questions we may not have covered
